app/cells/select/select_cell.rb

```diff
@@ -0,0 +1,48 @@
+class SelectCell < FormCellBase
+  attribute :form, mandatory: true, description: 'A form object.'
```

Comment:
We were considering this but wanted to proceed one step at a time. It's on the roadmap... ;)